feature/home/src/main/java/com/epmedu/animeal/home/di/HomePresentationModule.kt

```diff
@@ -97,7 +112,7 @@ internal object HomePresentationModule {
     @ViewModelScoped
     @Provides
     fun providesFeedingHandler(
```

Comment:
Can we also move providers to shared feature, since we are going to reuse it?

Reply:
Done